src/utils/routes.ts

```diff
@@ -15,7 +15,7 @@ export function routeOptions(
     return {
         search: {
             ...search,
-            chain: typeof chain === 'string' ? chain : getSymbolicChainName(chain),
+            chain: typeof chain === 'string' ? chain : getChainKey(chain),
```

Comment:
If this PR gets live w/o the other one (slug) then this line changes amoy links. I'll get fixed by the other PR (#1976).